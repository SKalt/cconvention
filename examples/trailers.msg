feat(thing)!: ...

body thing thing thing thing
thing thing thing thing thing
thing

BREAKING CHANGE: detailed description
BREAKING-CHANGE: also valid
Signed-off-by: Dev Eloper <devel@example.com>
# TODO: multiline-trailer: a
#  b
#  c

