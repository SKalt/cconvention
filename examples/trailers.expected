examples/trailers.msg:8:1	Warning	footer_leading_blank	Missing blank line before trailers.

